### PR TITLE
K8SPXC-567 update crash recovery doc and add relevant option

### PR DESCRIPTION
**Pull request by fiowro**
[K8SPXC-567](https://jira.percona.com/browse/K8SPXC-567) [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

File: source/recovery.rst [hunk 1]
```diff
@@ -19,35 +19,30 @@ continuously restarting, and generates the following errors in the log::
...
 
-Bootstrap Crash Recovery method
--------------------------------
+Automatic Crash Recovery
+------------------------
 
-In this case recovery is done automatically. The recovery is triggered by the
-``pxc.forceUnsafeBootstrap`` option set to ``true`` in the ``deploy/cr.yaml``
-file::
+Recovery is done automatically if the ``spec.autoPXCRecovery`` option set to
```
Comment (hors) [inline source/recovery.rst:31]: It is by default 'true', even if this option does not exist or is commented in CR.

File: source/operator.rst [hunk 2]
```diff
@@ -79,6 +79,10 @@ main subsystems of the cluster:
      - ``SmartUpdate``
      - A strategy the Operator uses for :ref:`upgrades<operator-update>`
 
+   * - autoPXCRecovery
```
Comment (tplavcic) [inline source/operator.rst:82]: If I see correctly this is named differently and it goes under `pxc` section. The option is called `autoRecovery: true`

File: source/recovery.rst (hunk 1 above)
Comment (tplavcic) [inline source/recovery.rst:31]: `spec.pxc.autoRecovery `

File: source/operator.rst [hunk 3]
```diff
@@ -79,6 +79,10 @@ main subsystems of the cluster:
...
+     - boolean
+     - ``true``
+
 .. _operator.upgradeoptions-section:
```
Comment (tplavcic) [inline source/operator.rst:86]: seems we are missing `haproxy` section in the top of this page in the table

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: source/recovery.rst (hunk 1 above)
Comment (hors) [reply source/recovery.rst:31]: And yes, not it is `spec.pxc.autoRecovery`

Comment (fiowro) [reply source/recovery.rst:31]: Ok, so pxc.autoRecovery is true by default. 
Currently, I see ``#    autoRecovery: false`` in deploy/cr.yaml. Should it be fixed there?

File: source/operator.rst (hunk 3 above)
Comment (fiowro) [reply source/operator.rst]: I already see it in pxc-docs branch, perhaps it was brought there in different PR

File: source/recovery.rst [hunk 4]
```diff
@@ -19,35 +19,31 @@ continuously restarting, and generates the following errors in the log::
...
 
-Bootstrap Crash Recovery method
--------------------------------
+Automatic Crash Recovery
+------------------------
 
-In this case recovery is done automatically. The recovery is triggered by the
-``pxc.forceUnsafeBootstrap`` option set to ``true`` in the ``deploy/cr.yaml``
-file::
+Crash recovery can be done automatically. This b behavior is controlled by the
```
Comment (tplavcic) [inline source/recovery.rst:31]: `b behavior` -> `behavior`

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Review by hors — CHANGES_REQUESTED** [review]
@fiowro Now this doc is valid only for  PXCO <= 1.6.0 , So, please revert to https://github.com/percona/percona-xtradb-cluster-operator/commit/9e88b375f9cd1e6bdb836493e0d029f58364f21b commit

**Review by hors — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**